layers/prism.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2015 Google, Inc. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+// http://www.tcpdump.org/linktypes/LINKTYPE_IEEE802_11_PRISM.html
+
+package layers
+
+import (
+	"encoding/binary"
+
+	"github.com/google/gopacket"
+)
+
+const ()
+
+func decodePrismValue(data []byte, pv *PrismValue) {
+	pv.Did = binary.LittleEndian.Uint32(data[0:4])
+	pv.Status = binary.LittleEndian.Uint16(data[4:6])
+	pv.Length = binary.LittleEndian.Uint16(data[6:8])
+	pv.Data = binary.LittleEndian.Uint32(data[8:12])
+}
+
+type PrismValue struct {
+	Did    uint32
```

Comment:
Also, DID should probably be an enum.  Maybe something like:

type PrismDID uint32
const (
  PrismHostTime PrismDID = X
  PrismMACTime PrismDID = Y
  ...
)